recipes-kernel/linux/linux-beaglev_5.10.bb

```diff
@@ -0,0 +1,23 @@
+SUMMARY = "An example kernel recipe that uses the linux-yocto and oe-core"
```

Comment:
I think this file should be called 'linux-starfive*'

Reply:
renamed